pkg/gadgets/trace/bind/tracer/tracer.go

```diff
@@ -26,11 +26,12 @@ import (
 	"github.com/cilium/ebpf/link"
 	"github.com/cilium/ebpf/perf"
 	"github.com/vishvananda/netlink"
-
+	
```

Comment:
Please be sure to run `make lint` for format this file. (remove trailing while spaces on this case)